utils/Makefile

```diff
@@ -62,45 +63,45 @@ viewcover:
 # Conditional virtualenv building strategy taken from this great post
 # by Marcel Hellkamp:
 # http://blog.bottlepy.org/2012/07/16/virtualenv-and-makefiles.html
-venv: oo-installenv/bin/activate
-oo-installenv/bin/activate: test-requirements.txt
+$(VENV): $(VENV)/bin/activate
+$(VENV)/bin/activate: test-requirements.txt
 	@echo "#############################################"
 	@echo "# Creating a virtualenv"
 	@echo "#############################################"
-	test -d venv || virtualenv $(NAME)env
-	. $(NAME)env/bin/activate && pip install setuptools==17.1.1
-	. $(NAME)env/bin/activate && pip install -r test-requirements.txt
-	touch $(NAME)env/bin/activate
+	test -d $(VENV) || virtualenv $(VENV)
+	. $(VENV)/bin/activate && pip install setuptools==17.1.1
```

Comment:
Out of curiosity, do you know why are we pinning the `setuptools` version to this particular version?